src/screens/AddNotificationEmailScreen.tsx

```diff
@@ -34,6 +34,7 @@ interface Props {
   error: string;
```

Comment:
I think there should be a new screen for ChangeEmail, this screen is only for adding, let's not try to do everything on one screen, the change email flow is different than add because of different amount of fields, different validation, different actions on submit, different variables available. 
It's missing the validation if the new email is the same as the old one.
Update flow uses modify endpoint

Reply:
there are exactly the same except for the 'old email' label. What's the point of maintaining two screens with identical styling, almost the same UI, and the same redux stuff?

Reply:
it's not only a label but also different validation (checking if the email is already subscribed), different variables available (we have an email address in that screen), and different actions, we use "modify" action, so in fact, the only thing that is the same the email input. Check that you had to add a custom function onSuccessWhileChanging that is only used for change flow because we only then have email available. This is the sign that we try to do too much on one screen

Reply:
done